Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -1455,34 +1519,21 @@ - (void)keyboardWillHideNotification:(NSNotification *)notification
     contentInset.bottom = 0.f;
     self.collectionView.contentInset = contentInset;
     self.collectionView.scrollIndicatorInsets = contentInset;
-    [self.emptyViewBottomConstraint setConstant:0.f];
+    self.currentKeyboardHeight = 0.f;
+    [self.emptyViewBottomConstraint setConstant:self.currentKeyboardHeight];
```

Comment:
[Nit] Just for the sake of consistency 😄 
```suggestion
    [self.emptyViewBottomConstraint setConstant:-self.currentKeyboardHeight];
```

Reply:
Resolved here https://github.com/wordpress-mobile/MediaPicker-iOS/commit/e54f6a1a818e1226233fcbca6bb9649e57447b42 😄